woocommerce/google-listings-and-ads
# Integrate frontend data store with the changed `mc/countries` API

### Changes proposed in this Pull Request:

This PR implemented the **Integrate with changed mc/countries API** part in #1356

- Rename the selector, resolver and reducer of the original `getCountries` in wp-data, and adjust them to get Merchant Center supported countries and continents from API. The relevant hook `useGetCountries` was renamed as well.
   - The reason for renaming is that it returns the countries supported by the Merchant Center, not the countries supported by WC or all countries.
- Replace all `getCountries` and `useGetCountries` uses with `useMCCountries`.

### Screenshots:

![image](https://user-images.githubusercontent.com/17420811/159904499-77069911-04eb-4438-a6ba-806bd74bcde6.png)

![image](https://user-images.githubusercontent.com/17420811/159904556-08a068bf-9ac0-445a-8944-61e2fa161e34.png)

### Detailed test instructions:

1. Go to step 1 in the free listings edit page.
2. Check if the countries in the audience selectors are the same as before. It should have 93 countries.

### Changelog entry
